client/src/App.test.js

```diff
@@ -1,9 +1,43 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
+import DataPullClass from './components/DataPullClass';
+import DisplayPlayers from './components/DisplayPlayers';
+import Button from './components/Button'
+import { render, getByText, fireEvent, queryAllByTestId, queryAllByText } from '@testing-library/react';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
-  ReactDOM.unmountComponentAtNode(div);
+test('renders App without crashing', () => {
+  render(<App />);
 });
+
+test('renders DataPullClass without crashing', () => {
+  render(<DataPullClass />);
+});
+
+test('checks if names don"t have numbers', () => {
+  const {container} = render(<DataPullClass />);
+  const Players = queryAllByTestId(container, 'player')
+  Players.forEach(player => {
+    expect(player.textContent).toBeNaN();
```

Comment:
Is there anything being rendered in this component when the tests run since there is no data in it until the fetch call sets it?